l10n_it_ricevute_bancarie/models/account.py

```diff
@@ -38,6 +38,25 @@ class ResPartnerBankAdd(models.Model):
         help="Identification Code of the Company in the Interbank System.",
     )
 
+    def unlink(self):
+        protected_records = (
+            self.env["account.move"]
+            .search([("riba_partner_bank_id", "in", self.ids)])
+            .mapped("riba_partner_bank_id")
+        )
+        records_to_delete = self - protected_records
```

Comment:
```suggestion
        records_to_delete = self - protected_records
```

non ho capito questa, non dovrebbe essere necessario, sbaglio?